homeassistant/components/jlrincontrol/binary_sensor.py

```diff
@@ -2,7 +2,7 @@
 import logging
 
 from homeassistant.components.binary_sensor import BinarySensorDevice
-from homeassistant.components.jlrincontrol import JLREntity, RESOURCES
+from . import JLREntity, RESOURCES
```

Comment:
I'd check if this is allowed as per HomeAssistant requirements. Obviously syntactically its fine, but it might not be with regards to linting reqs.

Reply:
i see "relative import" with dot undet great migration here:
https://developers.home-assistant.io/blog/

Reply:
Ah ok perfect. Looks like it's allowed them 🙂